recipes/skan/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "skan" %}
+{% set version = "0.1.2" %}
+{% set sha256 = "98e60bc948527b7a60ca8864757d12b87d04dc2c2df7561dab03c1d33a052f04" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip : True  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.11
+    - scipy >=0.18
+    - networkx >=1.11
+    - pandas
+    - numba
```

Comment:
@synapticarbors yeah I dunno! I am inclined to put them in, since I haven't tested with anything older. (Nor do I intend to. =P) But I'm happy to heed advice, let me know what you think.

Thanks for reviewing!